### ISSUE TITLE
Latest JMdict entries missing?

### ISSUE THREAD

**Issue by fasiha** [Doublevil/JmdictFurigana#13]
Picking up from #10, where you said

> fwiw in the latest release I tried using the most recent JMdict XML file and it gave me the exact same number of entries.

I am wondering about this because I recently submitted 作り込む to JMdict and (unsurprisingly) it's not in the latest JmdictFurigana release. Other recent additions to JMdict that are currently missing—

- 委蛇 (added 2020-09-01)
- 天埋 (2020-08-31)
- 夏萩 (2020-08-12, which I think is before you released version 2.2)

As an aside, a project I use alongside JmdictFurigana is [JMdict Simplified](https://github.com/scriptin/jmdict-simplified) whose author [recently](https://github.com/scriptin/jmdict-simplified/issues/14) set up GitHub Actions (which has Windows hosts in the free tier) to automatically rebuild the project every fifteen days. I absolutely don't want to put any pressure or stress on you to investigate this, but it's one option to keep JmdictFurigana automatically up-to-date with JMdict.

As always, thank you for your hard work!

**Comment by Doublevil** [comment]
My bad. I don't know what I did wrong last time I checked, but I tried updating the file again and this time I got more entries as expected. Thanks for raising this issue.
I'm going to release an updated version asap.

I'm also going to look into setting up an automated build and release system through GitHub Actions (it does look pretty cool). This will imply a few other things, like upgrading the project to .net core, and having the program download the latest dictionary files instead of just reading them from the disk, which is for the best anyways.
